apps/dashboard/src/hooks/useImageFileOrUrl.ts

```diff
@@ -1,15 +1,16 @@
 import { useMemo } from "react";
+import type { FileOrBufferOrString } from "thirdweb/storage";
 import { isBrowser } from "utils/isBrowser";
 
-export function useImageFileOrUrl(image?: string | File): string {
+export function useImageFileOrUrl(image?: FileOrBufferOrString): string {
```

Comment:
why was the prior typing not working?

Reply:
I reverted the change back to `main` and it IS working lol
Last night it was throwing some hard-to-read error which led me to believe I have to use the new type that I exported, for consistency.

(Now that this change is reverted I will close this)